stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/options/internal/AttachmentGalleryOptionsDialogFragment.kt

```diff
@@ -71,42 +106,87 @@ internal class AttachmentGalleryOptionsDialogFragment : FullScreenDialogFragment
         _binding = null
     }
 
+    /**
+     * Sets a callback for the "show in chat" option.
+     *
+     * @param showInChatOptionHandler The handler to set.
+     */
+    fun setShowInChatOptionHandler(showInChatOptionHandler: AttachmentOptionHandler) {
+        this.showInChatOptionHandler = showInChatOptionHandler
+    }
+
+    /**
+     * Sets a callback for the "reply" option.
+     *
+     * @param replyOptionHandler The handler to set.
+     */
+    fun setReplyOptionHandler(replyOptionHandler: AttachmentOptionHandler) {
+        this.replyOptionHandler = replyOptionHandler
+    }
+
+    /**
+     * Sets a callback for the "delete" option.
+     *
+     * @param deleteOptionHandler The handler to set.
+     */
+    fun setDeleteOptionHandler(deleteOptionHandler: AttachmentOptionHandler) {
+        this.deleteOptionHandler = deleteOptionHandler
+    }
+
+    /**
+     * Sets a callback for the "save image" option.
+     *
+     * @param saveImageOptionHandler The handler to set.
+     */
+    fun setSaveImageOptionHandler(saveImageOptionHandler: AttachmentOptionHandler) {
+        this.saveImageOptionHandler = saveImageOptionHandler
+    }
+
+    /**
+     * Set the message ownership.
+     *
+     * @param isMineMessage If the message belongs to the current user.
+     */
+    fun setIsMineMessage(isMineMessage: Boolean) {
```

Comment:
Nitpicky comment, but maybe `isMineMessage` should be renamed to `isMessageMine` as the first doesn't make sense gramatically.